cpp/net/connection_pool.cc

```diff
@@ -205,7 +234,15 @@ evhtp_res ConnectionPool::Connection::ConnectionErrorHook(
     // If someone hasn't already modified the default status, set it to a
     // generic "something went wrong" value here:
     if (conn->request->status == 200) {
-      conn->request->status = kUnknownErrorStatus;
+      VLOG(1) << "error flag (0x" << std::hex << static_cast<int>(flags)
+              << "):" << ErrorFlagDescription(flags);
```

Comment:
I think it affects any scalar values sent to the stream after the manipulator, it won't affect the string description though.